styles/desktop-version.css

```diff
@@ -0,0 +1,632 @@
+    grid-column: 1;
+    grid-row: 1 / 3;
+    width: 100%;
+    height: auto;
+    object-fit: fill;
+  }
+
+  .image-after-started-image {
+    display: initial;
+    grid-column: 6 / 8;
+    grid-row: 5 / 7;
+    width: 100%;
+    height: auto;
+    object-fit: cover;
+    align-self: end;
+    justify-self: start;
+  }
+
+  .get-started-section {
+    grid-column: 2 / 7;
+    grid-row: 1 / 7;
+    margin-left: -5vw;
+  }
+
+  .get-started-background > section > h1 {
+    transform: none;
+    margin-top: 10vh;
+    margin-bottom: 4vh;
+  }
+
+  .get-started-background > .get-started-section > .section-title {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 80px;
+    text-align: center;
+    padding: 0;
+  }
+
+  .get-started-background > .get-started-section > .section-title > .greetings {
+    font-style: normal;
+    font-weight: 800;
+    font-size: 14px;
+    line-height: 21px;
+  }
+
+  .form-message {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto;
+    gap: 8%;
+  }
+
+  .yours {
+    border: none;
+  }
+
+  .yours label {
+    width: 100%;
+    color: #7d7c84;
+    background: none;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 15px;
+    padding-top: 5px;
+    text-align: start;
+    padding-left: 5%;
+    border: none;
+  }
+
+  .yours input,
+  .yours textarea {
+    width: 100%;
+    background: #fff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 15px;
+    padding-top: 5%;
+    padding-bottom: 5%;
+    padding-left: 5%;
+    margin-left: 5%;
+    margin-right: 5%;
+    border-top: 1px solid rgb(173, 171, 171);
+    border-right: 1px solid rgb(173, 171, 171);
+    border-bottom: 1px solid rgb(173, 171, 171);
+    border-left: 1px solid rgb(173, 171, 171);
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+  }
+
+  .yours > .label-textarea-mobile {
+    display: none;
+  }
+
+  .yours > label.label-textarea-desktop {
+    display: initial;
+  }
+
+  #textarea {
+    grid-column: 1 / 3;
+    grid-row: 2;
+    width: 219%;
+    margin-left: 4%;
+    margin-right: 920%;
+    border-radius: 5px;
+    margin-bottom: 15%;
+  }
+
+  ::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+    font-family: 'Poppins', sans-serif;
+    font-size: 15px;
+    font-weight: 400;
+    color: rgb(29, 28, 28);
+    text-align: start;  /* Firefox */
+  }
+
+  ::-ms-input-placeholder { /* Microsoft Edge */
+    font-family: 'Poppins', sans-serif;
+    font-size: 15px;
+    font-weight: 400;
+    color: rgb(0, 0, 0);
+    text-align: start;
+  }
+
+  :-ms-input-placeholder { /* Internet Explorer 10-11 */
+    font-family: 'Poppins', sans-serif;
+    font-size: 15px;
+    font-weight: 400;
+    color: rgb(0, 0, 0);
+    text-align: start;
+  }
+
+  .button-get-started {
+    width: 50%;
```

Comment:
- [ ] I think you may not necessarily use a relative with for the contact form button element, kindly use `254px` (for the desktop version) as suggested earlier according to the given template ℹ️

Reply:
fixes done